### ISSUE TITLE
Angular

### ISSUE THREAD

**Issue by asaf-paz** [jimmywarting/FormData#48]
Hi,
I've tried to use your polyfill on my Angular app (version 5 as of now) and I had partial success.
It works great on Android devices, Chrome and IE on Windows 10. but on IOS IPhone it did not go well and also on Internet Explorer version 11 on Windows 7 had similar issue.
The functions has() , append() etc. work well.
The issue is with the post action. 
There i get an error.
I've been trying to get my head around it and had no luck. 
Can you give me an advise maybe?
Thanks.

**Comment by jimmywarting** [comment]
Could you try a minimal example using no angular?
Would help with some form of stacktrace of what is going wrong or some dump on the request that was mad in the network inspector

test if calling `fd._blob()` works

**Comment by asaf-paz** [comment]
Looks like the polyfill is working great. I think that it has something to
do with our infrastructure.
I think I found the reason why its not working.
Our application sits behind some sort of firewall and reverse proxy (TMG
and UAG).
And i think that because the Content-Type string that is being sent in the
header, it somehow blocks the post.

I noticed that on chrome browser the content type is of this sort:
Content-Type:multipart/form-data;
boundary=----WebKitFormBoundarybBLw14JY2tZ7zSL6

and on other browsers that use the formdata polyfill, the header looks like
this:
Content-Type: multipart/form-data;
boundary=----formdata-polyfill-0.06878081017569748

Maybe its the dash character, or maybe its the dot that appears after it.
I'm guessing that the firewall is filtering the post.

Do you know if there is a way I can change the boundary string and set it
to something else manually?

Thanks a lot!

**Comment by jimmywarting** [comment]
Could maybe change it to something similar to webkits. Was changed before don't remember the reason but need to look it up among the issues

**Comment by jimmywarting** [comment]
One requirement is that it's lowercased... #17
Dose removing the dot solve the issue or should we get rid of the dashes to?

**Comment by asaf-paz** [comment]
Sorry.. It still didn’t help.
When I do the POST action using Microsoft Edge or using any browser on IPhone I get an error. When I examined the response body the error I got was an actual web page with 404 error. It must be our firewall that is causing that issue, although our system people don’t see anything in their logs. I’ve tried changing the string to imitate Webkit content type header but as you stated below, it is still not exactly the same because of the lowercase thingy...
I think I have no choice but to use the old fashion way using regular post by changing the photos to base64 on client side and do a regular plaintext post.
What a pain ... Because your polyfill is really working great...
Thanks a lot anyways. 

‫ב-2 בינו׳ 2018, בשעה 21:03, ‏‏Jimmy Wärting ‏<notifications@github.com> כתב/ה:‬
